.github/workflows/beta.yml

```diff
@@ -1,6 +1,14 @@
 name: Beta release CLI
 on:
   workflow_dispatch:
+    inputs:
+      version:
+        type: string
+        description: The version to release
+        default: 'v0.8.0-beta'
```

Comment:
I would remove the default value in this workflow, and make it mandatory.

We should also update the contribution guide